File: src/server/Mimirorg.TypeLibrary/Models/Client/SimpleLibCm.cs
using Mimirorg.TypeLibrary.Enums;

namespace Mimirorg.TypeLibrary.Models.Client
{
    public class SimpleLibCm
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Iri { get; set; }
        public ICollection<TypeReferenceCm> TypeReferences { get; set; }
        public string Description { get; set; }
        public State State { get; set; }
        public DateTime Created { get; set; }
        public string CreatedBy { get; set; }
        public ICollection<AttributeLibCm> Attributes { get; set; }
        public string Kind => nameof(SimpleLibCm);
    }
}

File: src/client/src/models/tyle/client/simpleLibCm.ts
import { AttributeLibCm } from "./attributeLibCm";

export interface SimpleLibCm {
  id: string;
  name: string;
  iri: string;
  contentReferences: string[];
  description: string;
  created: string;
  createdBy: string;
  attributes: AttributeLibCm[];
  kind: string;
}
